app/models/athlete.rb

```diff
@@ -0,0 +1,11 @@
+class Athlete < ApplicationRecord
+  validates_presence_of :name, :gender, :birthday
+  validates_numericality_of :height, :weight
+  
+
+  belongs_to :user
+  has_many :movements, dependent: :destroy
+  has_many :movement_goals, through: :movements
+  has_many :goals, dependent: :destroy
+  has_many :progresses, dependent: :destroy
+end
```

Comment:
I think this should be

belongs_to :user
has_many :progress_movements, dependent: :destroy
has_many :progresses, through: :progress_movements
has_many :movement_goals, dependent: :destroy
has_many :goals, through: :movement_goals